app/components/cart/cartView.html

```diff
@@ -0,0 +1,46 @@
+<div>
+    <nav-bar/>
+</div>
+
+<div class="container-fluid">
+    <div class="row">
+        <table class="table table-hover table-bordered">
+            <thead class="bg-primary">
+            <tr class="text-light">
+                <th scope="col"></th>
+                <th scope="col"></th>
+                <th scope="col">Price</th>
+                <th scope="col">Quantity</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr ng-repeat="item in cartItems track by item.productId">
+                <td>
+                    <div class="row justify-content-center">
+                        <img ng-src="{{item.product.imagePosterUrl}}" alt="" class="img-fluid">
+                    </div>
+                </td>
+                <td>
+                    <div class="pl-2">
+                        <div class="row">
+                            <h4>{{item.product.name}}</h4>
+                        </div>
+                        <div class="row mt-1">
+                            <h6 class="text-success" ng-show="item.inStock">In Stock</h6>
+                            <h6 class="text-danger" ng-hide="item.inStock">Out Of Stock</h6>
+                        </div>
+                        <div class="row mt-1">
+                            <a href="" ng-click="removeItem(item)">Delete</a>
```

Comment:
remove empty href attribute

Reply:
I put it to make the delete button have the hand pointer instead of regular one